packages/opentelemetry-api/src/api/context.ts

```diff
@@ -20,12 +20,18 @@ import {
   Context,
 } from '@opentelemetry/context-base';
 
+const NOOP_CONTEXT_MANAGER = new NoopContextManager();
+
+const GLOBAL_CONTEXT_MANAGER_API_KEY = Symbol.for(
```

Comment:
I think we should add a section in top README and explain how we achieve/handle a global API thing.

Reply:
README is visible to users in our docs. Is this necessary or will it just confuse them? We can probably add an explainer that the versions must match or the API will return a noop, but the internal mechanism how it does that should be not important to users. Maybe a comment at the top of global utils?

Reply:
I added an explainer to the README